internal/cri/opts/container.go

```diff
@@ -76,6 +76,11 @@ func WithVolumes(volumeMounts map[string]string, platform imagespec.Platform) co
 		}
 		mounts = mount.RemoveVolatileOption(mounts)
 
+		// Always copy files without UID/GID transformations.
+		// The ID transformations are only needed when running inside a userns, that is not
+		// the case here.
+		mounts = mount.RemoveIDMapOption(mounts)
```

Comment:
@mikebrow do we still need to support that volume defined in image? 
Kubelet always recreates a container once the current one exits. Using an image volume only adds unnecessary I/O.
If there is no valid use case for it, we probably consider deprecating it.